README.md

```diff
@@ -0,0 +1,125 @@
+# Jaworsi-Lab Napari Plugin
+
+This [Napari](https://napari.org) plugin streamlines the process of segmenting, analyzing, and counting cells in 3D datasets. Designed with researchers in mind, it provides an intuitive interface to handle bioimaging data, offering seamless integration with the Jaworski Lab datasets and their acquisition hardware.
+
+## Features
+
+- **Load BioFormat Files**: Load BioFormat Files Easily load .oir files created by lab acquisition hardware (e.g., Imaris), ensuring compatibility and a smooth workflow for researchers.
+- **Preprocess Data**: Adjust contrast and apply a Gaussian filter to enhance the dataset, followed by thresholding to create binary masks. This step improves the segmentation and labeling of cells for subsequent analysis.
+- **Count Cells in Dataset**: Count the number of cells present by combining the enhanced dataset with binary maps. The plugin enables users to focus on regions of interest and generate accurate cell counts.
+
+
+## User's manual
+
+### Installation
+
+Install [napari](https://napari.org/dev/tutorials/fundamentals/installation.html)  and then install this plugin using the following command
+
+
+### How to use
+
+1. Run napari from your python environment
+2. Select the Plugin option located at the top menu bar
+3. Select Jaworski lab cell plugin
+
+   <img src="images/README/napari_img.png" alt="Image description" width="500" height="300">
+4. After a few seconds the widgets will be added at the right side of the napari window
+   
+   <img src="images/README/plugin_img.png" alt="Image description" width="200" height="300">
+
+5. The following image illustrates the series of steps to segment, label, and count cells in the dataset.
+   <img src="images/README/pipeline.png" alt="Image description" width="630" height="200">
+
+### Load dataset
+
+The top widget includes a button to open a file selection window. Click on "Select File," choose the .oir dataset, and then click the "Open BioImage" button in the widget.
+   
+   <img src="images/README/load_dataset_widget.png" alt="Image description" 
+   width="280" height="160">
+
+
+### Pre Process data
+
+This widget allows you to modify various values to adjust the contrast of the dataset and apply a Gaussian filter. By default, we have set values that work best for the Jaworski lab datasets. You don’t need to modify these values, but feel free to adjust them if needed. Once ready, press the "Pre-process BioImage" button, and additional layers will be added to the visualizer.
```

Comment:
I would avoid using "widget" to describe sections of your plugin. To a user it's not clear  where one widget starts and ends, or what a widget actually is. 

Perhaps:
You may pre-process the image with a Gaussian filter and contrast adjustments. Default values are set to optimal values for the Jaworski lab datasets. Once values are to your preference, click "Pre-process BioImage" and additional layers will be added to the visualizer.